core/post.ts

```diff
@@ -66,6 +67,13 @@ export function deletePost(postId: string): Promise<void> {
   return deleteSyncMapById(getClient(), Post, postId)
 }
 
+export async function changePost(
```

Comment:
Can you add a simple test for that? You can just add a few lines in existing test.